quiche/src/recovery/mod.rs

```diff
@@ -1525,6 +1537,201 @@ mod tests {
         );
     }
 
+    // TODO: `congestion` and `gcongestion` behave differently. That might be ok
+    // given the different algorithms but it would be ideal to merge and share
+    // the logic.
+    #[rstest]
+    fn relaxed_thresholds_on_reordering(
+        #[values("bbr2_gcongestion")] cc_algorithm_name: &str,
+    ) {
+        let mut cfg = Config::new(crate::PROTOCOL_VERSION).unwrap();
+        cfg.enable_relaxed_loss_threshold = true;
```

Comment:
Worth considering parametrizing the existing test with `#[values(false, true)] enable_relaxed_loss_threshold` ?

It is currently hard to tell how this very long test differs in behavior for the two cases.